tools/tests/README.md

```diff
@@ -126,15 +124,11 @@ el.innerHTML = template;
 document.body.appendChild(el);
 ````
 
-If you do this multiple times within a test, it is good practice
-to clean the DOM before attaching a new element to avoid duplications:
+If you do this multiple times within a test, it is good practice to clean the DOM before attaching a new element to avoid duplications. Call the ``

```

Comment:
```suggestion
If you do this multiple times within a test, it is good practice to clean the DOM before attaching a new element to avoid duplications. Call the
```

Reply:
Fixed.